docs/config/README.md

```diff
@@ -977,13 +977,14 @@ The module will be shown when inside a nix-shell environment.
 
 ### Options
 
-| Variable     | Default      | Description                        |
-| ------------ | ------------ | ---------------------------------- |
-| `use_name`   | `false`      | Display the name of the nix-shell. |
-| `impure_msg` | `"impure"`   | Customize the "impure" msg.        |
-| `pure_msg`   | `"pure"`     | Customize the "pure" msg.          |
-| `style`      | `"bold red"` | The style for the module.          |
-| `disabled`   | `false`      | Disables the `nix_shell` module.   |
+| Variable     | Default       | Description                                       |
+| ------------ | ------------- | ------------------------------------------------- |
+| `use_name`   | `false`       | Display the name of the nix-shell.                |
+| `impure_msg` | `"impure"`    | Customize the "impure" msg.                       |
+| `pure_msg`   | `"pure"`      | Customize the "pure" msg.                         |
+| `style`      | `"bold blue"` | The style for the module.                         |
+| `symbol`     | `"❄️  "`       | The symbol used before displaying the shell name. |
```

Comment:
In general we try to keep the `style` and `disabled` variables at the bottom of the table.

```suggestion
| `symbol`     | `"❄️  "`       | The symbol used before displaying the shell name. |
| `style`      | `"bold blue"` | The style for the module.                         |
```

Reply:
Fixed, thanks!